network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/image_teaser_block.html

```diff
@@ -0,0 +1,22 @@
+{% extends "./base_streamfield_block.html" %}
+{% load l10n wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+  {% with card=value %}
+    <section class="tw-grid tw-grid-cols-1 large:tw-grid-cols-2 tw-gap-5 large:tw-gap-[60px] tw-my-4 large:tw-mb-7">
+      <div class="tw-row-start-2 large:tw-row-start-1">
+        <h2 class="tw-mb-4">{{ card.title }}</h2>
+        {{ card.text|richtext }}
+        {% if card.url and card.url_label %}
+          <a class="tw-{{ card.styling }} link-button tw-mt-4" href="{{ card.url }}">
+            {{ card.url_label }}
+          </a>
+        {% endif %}
+      </div>
+      <div class="tw-row-start-1 large:tw-row-start-1">
```

Comment:
What do you think about removing these classes? I think the grid elements would default to start in the first row and the override for the large breakpoint sets the same value.

Reply:
Yep good catch, i've removed these now